WrapImage/nifti_wrapper.py

```diff
@@ -27,11 +30,11 @@ def read_json_file(json_file):
 
     return json_data
 
-def save_nifti_3d(data, output_file):
+def save_nifti_3d(data, output_file, **kwargs):
     """
     For saving the 3d nifti images of the output of the algorithm
     """
-    output_img = nib.Nifti1Image(data, np.eye(4))
+    output_img = nib.nifti1.Nifti1Image(data, np.eye(4), **kwargs)
```

Comment:
The `np.eye(4)` should also be an input argument, but the default should be `np.eye(data.ndim+1)` I think